public/migrate.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Test page for hitting the API</title>
+    <style>
+      form fieldset { border: none; margin: 0; padding: 0; }
+      form fieldset + fieldset { margin-top: 0.5em;}
+      form span { display: inline-block; width: 10em; }
+    </style>
+  </head>
+  <body>
+    <h1>A test post to http://test.example.com:8000/entries</h1>
+
+    <form name="entries" method="POST">
+      <fieldset>
+        <label>
+          <div>JSON array of entries</div>
+          <textarea style="width: 100%" rows=30 id="entryArray" name="entryArray">
+          </textarea>
+        </label>
+      </fieldset>
+      <!-- 'featured' is set through administration -->
+      <fieldset>
+        <button id="submit" disabled="disabled">submit</button>
+        <div>Can't submit? Check your syntax</div>
+      </fieldset>
+    </form>
+
+  <script>
+
+    var currentNonce,
+        currentCSRF,
+        currentUser;
+      
+    var entries;
+    
+    //Sample:
+        //[{
+        //     "title": "test full entry 1",
+        //     "description": "description full entry",
+        //     "tags": ["tag1", "tag2"],
+        //     "interest": "interest field",
+        //     "get_involved": "get involved text field",
+        //     "get_involved_url": "http://example.com/getinvolved",
+        //     "thumbnail_url": "http://example.com/",
+        //     "content_url": "http://example.com/",
+        //     "internal_notes": "Some internal notes",
+        //     "featured": true,
+        //     "issues": ["Decentralization"],
+        //     "creators": ["Pomax", "Alan"]
+        // },
+        // {
+        //     "title": "test full entry 2",
+        //     "description": "description full entry",
+        //     "tags": ["tag1", "tag2"],
+        //     "interest": "interest field",
+        //     "get_involved": "get involved text field",
+        //     "get_involved_url": "http://example.com/getinvolved",
+        //     "thumbnail_url": "http://example.com/",
+        //     "content_url": "http://example.com/",
+        //     "internal_notes": "Some internal notes",
+        //     "featured": true,
+        //     "issues": ["Decentralization"],
+        //     "creators": ["Pomax", "Alan"]
+        // }];
+
+    // what to do when "submit" is clicked
+    var submitForm = () => {
+      document.querySelector("#submit").setAttribute("disabled", "disabled");
+      entries = JSON.parse(document.getElementById("entryArray").value);
+      getNonce(postAnEntry);
+    };
+
+    // don't let the browser submit this form. We'll handle it.
+    document.forms.entries.onsubmit = () => false;
+    document.querySelector("#submit").onclick = submitForm;
+    var textBox = document.getElementById("entryArray");
+    var submitBtn = document.querySelector("#submit");
+    textBox.onkeyup = (e) => {
```

Comment:
No. What handler would you suggest? `onchange` seemed to depend on focus, so that wasn't great either.